apache-solr/README.md

```diff
@@ -0,0 +1,133 @@
+![Apache Solr Cluster Overview Dashboard 2](https://storage.googleapis.com/grafanalabs-integration-assets/apache-solr/screenshots/apache-solr-cluster-2.png)
+
+## Apache Solr Query Performance Overview
+
+The Apache Solr query performance overview dashboard provides details on various query load and latency, update handlers, cache, timeout and error metrics.
+
+![Apache Solr Query Performance Overview Dashboard 1](https://storage.googleapis.com/grafanalabs-integration-assets/apache-solr/screenshots/apache-solr-query-performance-1.png)
+![Apache Solr Query Performance Overview Dashboard 2](https://storage.googleapis.com/grafanalabs-integration-assets/apache-solr/screenshots/apache-solr-query-performance-2.png)
+
+## Apache Solr Resource Monitoring Overview
+
+The Apache Solr resource monitoring overview dashboard provides details on connections, threads, core FS usage, as well as JVM and Jetty metrics.
+
+![Apache Solr Resource Monitoring Overview Dashboard 1](https://storage.googleapis.com/grafanalabs-integration-assets/apache-solr/screenshots/apache-solr-resource-monitoring-1.png)
+![Apache Solr Resource Monitoring Overview Dashboard 2](https://storage.googleapis.com/grafanalabs-integration-assets/apache-solr/screenshots/apache-solr-resource-monitoring-2.png)
+
+## Apache Solr Logs Overview
+
+The Apache Solr logs overview dashboard provides details on slow requests, garbage collection, and error logs. [Promtail and Loki needs to be installed](https://grafana.com/docs/loki/latest/installation/) and provisioned for logs with your Grafana instance. The default Apache Solr error log path is `/var/solr/logs/solr.log` for each instance on Linux.
+
+Apache Solr logs are enabled by default in the `config.libsonnet` and can be removed by setting `enableLokiLogs` to `false`. Then run `make` again to regenerate the dashboard:
+
+```
+{
+  _config+:: {
+    enableLokiLogs: false,
+  },
+}
+```
+
+In order for the selectors to properly work for system logs ingested into your logs datasource, please also include the matching `job` and `solr_cluster` labels onto the [scrape_configs](https://grafana.com/docs/loki/latest/clients/promtail/configuration/#scrape_configs) as to match the labels for ingested metrics.
+
+```yaml
+scrape_configs:
+  - job_name: integrations/apache-solr
+    static_configs:
+      - targets: [localhost]
+        labels:
+          job: integrations/apache-solr
+          instance: '<your-instance-name>'
+          solr_cluster: '<your-cluster-name>'
+          __path__: /var/log/logs/*.log
+```
+
+![Apache Solr Logs Overview Dashboard](https://storage.googleapis.com/grafanalabs-integration-assets/apache-solr/screenshots/apache-solr-logs-overview.png)
+
+## Alerts Overview
+
+
+| Alert                                   | Summary                                                                                                             |
```

Comment:
I usually see this alerts section done as a simple list. Is this something we've done before? I don't have a problem with this way as long as it's something that has been accepted before/requested by Grafana.

Reply:
Yeah, this is how I have been doing it in the past with opensearch, meso, etc